drivers/wifi/eswifi/eswifi_core.c

```diff
@@ -520,7 +520,9 @@ int eswifi_mgmt_iface_status(const struct device *dev,
 	return 0;
 }
 
-static int eswifi_mgmt_scan(const struct device *dev, scan_result_cb_t cb)
+static int eswifi_mgmt_scan(const struct device *dev,
+			    struct wifi_scan_params *params,
+			    scan_result_cb_t cb)
```

Comment:
We should either add (for this and others below)

* `ARG_UNUSED(params)`
* or assuming most chipsets only support active (but this assumption might not be true)
```
if (params->scan_type != ACTIVE)
  return -ENOTSUP;
```

Reply:
Used ARG_UNUSED(params)